packages/config/src/genesisConfig/index.ts

```diff
@@ -142,6 +182,21 @@ function toHexStringNoPrefix(hex: string | Uint8Array): string {
   return strip0xPrefix(typeof hex === "string" ? hex : toHex(hex));
 }
 
-function computeForkDigest(currentVersion: Version, genesisValidatorsRoot: Root): ForkDigest {
-  return computeForkDataRoot(currentVersion, genesisValidatorsRoot).slice(0, 4);
+function computeForkDigest(
```

Comment:
there are tests for this on the spec [test_compute_fork_digest.py](https://github.com/ethereum/consensus-specs/blob/dev/tests/core/pyspec/eth2spec/test/fulu/validator/test_compute_fork_digest.py), but does not look like we have wired up the spec iterator for those yet